Sub-menu-via-event-target/test.js

```diff
@@ -0,0 +1,20 @@
+const ul = document.querySelector(".tree");
+
+const li = ul.querySelectorAll("li");
+
+li.forEach((item) => {
+  const span = document.createElement("SPAN");
+  span.classList.add("span");
+  item.prepend(span); // добавили перед тегом span
+  span.append(span.nextSibling); // добавили после тега span
+});
+
+ul.addEventListener("click", (event) => {
+  const children = event.target.closest("ul").querySelector("ul"); // у li, что мы нашли находим родителя, и у родителя ul внутри
```

Comment:
`.querySelector("ul")` 
Do you have a child ul?

Reply:
yes
![da](https://github.com/Konmly/Tasks/assets/62384781/4f7e0465-918c-43f9-a540-3288b94cbaef)